include/openthread/crypto.h

```diff
@@ -110,18 +106,18 @@ void otCryptoHmacSha256(const uint8_t *     aKey,
  * @param[out]    aTag           A pointer to the tag.
  *
  */
-void otCryptoAesCcm(const uint8_t *aKey,
-                    uint16_t       aKeyLength,
-                    uint8_t        aTagLength,
-                    const void *   aNonce,
-                    uint8_t        aNonceLength,
-                    const void *   aHeader,
-                    uint32_t       aHeaderLength,
-                    void *         aPlainText,
-                    void *         aCipherText,
-                    uint32_t       aLength,
-                    bool           aEncrypt,
-                    void *         aTag);
+void otCryptoAesCcm(otCryptoKey *aKey,
+                    uint16_t     aKeyLength,
```

Comment:
It seems key length is a part of `otCryptoKey`. This parameters seems redundant.

Reply:
Fixed.